pkg/util/util.go

```diff
@@ -399,3 +401,48 @@ func Zipit(source, target string) error {
 
 	return err
 }
+
+// DocExists checks whether an elasticsearch doc of a testrun exists in index testmachinery-* index
+func DocExists(log logr.Logger, esConfig *elasticsearch.Config, testrunID, testrunStartTime string) (docExists bool) {
+	log.Info("check if docs have already been ingested")
```

Comment:
```suggestion
	log.V(2).Info("check if docs have already been ingested")
```

I think the info not needed in every run